src/metabase/metabot.clj

```diff
@@ -95,16 +94,14 @@
   ([]
    "Show which card? Give me a part of a card name or its ID and I can show it to you. If you don't know which card you want, try `metabot list`.")
   ([card-id-or-name & _]
-   (let-404 [{card-id :id} (id-or-name->card card-id-or-name)]
-     (do-async (pulses/send-pulse! {:cards    [{:id card-id}]
-                                    :channels [{:channel_type   "slack"
-                                                :recipients     []
-                                                :details        {:channel *channel-id*}
-                                                :schedule_type  "hourly"
-                                                :schedule_day   "mon"
-                                                :schedule_hour  8
-                                                :schedule_frame "first"}]})))
-   "Ok, just a second..."))
+   (if-let [{card-id :id} (id-or-name->card card-id-or-name)]
+     (do
+       (do-async (let [attachments (pulse/create-and-upload-slack-attachments! [(pulse/execute-card card-id)])]
+                   (slack/post-chat-message! *channel-id*
+                                             nil
+                                             attachments)))
+       "Ok, just a second...")
+     (throw (Exception. "Not Found")))))
```

Comment:
Another argument for doing things the same way throughout the codebase is for ease of debugging. If Joe Rando comes in complaining that he keeps seeing "Not found" to debug it we can just stick a `printf` in the body of the `let-404` macro. Or perhaps add extra info to the exception info. If we always use `let-404`, every time we see a `Not found` we'll know where it came from!